src/lite-yt-embed.js

```diff
@@ -95,12 +102,53 @@ class LiteYTEmbed extends HTMLElement {
         LiteYTEmbed.preconnected = true;
     }
 
-    addIframe() {
+    fetchYTPlayerApi() {
```

Comment:
alsoo wondering if there's a lazy way to attach... https://github.com/paulirish/lite-youtube-embed/issues/82  hmmm!